che-editors.yaml

```diff
@@ -1,373 +1,442 @@
-version: 1.0.0
+version: 2.0.0
 editors:
-  - id: eclipse/che-theia/next
-    title: Eclipse Theia development version.
-    displayName: theia-ide
-    description: Eclipse Theia, get the latest release each day.
-    icon: https://raw.githubusercontent.com/theia-ide/theia/master/logo/theia-logo-no-text-black.svg?sanitize=true
-    repository: https://github.com/eclipse/che-theia
-    firstPublicationDate: "2019-03-07"
-    endpoints:
-      - name: "theia"
-        public: true
-        targetPort: 3100
-        attributes:
-          protocol: http
-          type: ide
-          secure: true
-          cookiesAuthEnabled: true
-          discoverable: false
-      - name: "webviews"
-        public: true
-        targetPort: 3100
-        attributes:
-          protocol: http
-          type: webview
-          secure: true
-          cookiesAuthEnabled: true
-          discoverable: false
-          unique: true
-      - name: "mini-browser"
-        public: true
-        targetPort: 3100
-        attributes:
-          protocol: http
-          type: mini-browser
-          secure: true
-          cookiesAuthEnabled: true
-          discoverable: false
-          unique: true
-      - name: "theia-dev"
-        public: true
-        targetPort: 3130
-        attributes:
-          protocol: http
-          type: ide-dev
-          discoverable: false
-      - name: "theia-redirect-1"
-        public: true
-        targetPort: 13131
-        attributes:
-          protocol: http
-          discoverable: false
-      - name: "theia-redirect-2"
-        public: true
-        targetPort: 13132
-        attributes:
-          protocol: http
-          discoverable: false
-      - name: "theia-redirect-3"
-        public: true
-        targetPort: 13133
-        attributes:
-          protocol: http
-          discoverable: false
-      - name: terminal
-        public: true
-        targetPort: 3333
-        attributes:
-          protocol: ws
-          type: collocated-terminal
-          discoverable: false
-          secure: true
-          cookiesAuthEnabled: true
-    containers:
+  - schemaVersion: 2.1.0
+    metadata:
+      name: eclipse/che-theia/next
+      displayName: theia-ide
+      description: Eclipse Theia, get the latest release each day.
+      icon: /v3/images/eclipse-che-logo.png
+      attributes:      
+        publisher: eclipse
+        version: next
+        type: Che Editor
+        title: Eclipse Theia development version.
+        category: Editor
```

Comment:
I think type and category can be removed as inside che-editors we expect to see editors ?